feature/src/adm.c

```diff
@@ -274,7 +274,7 @@ int compute_adm(const float *ref, const float *dis, int w, int h, int ref_stride
 	return ret;
 }
 
-int adm(int (*read_frame)(float *ref_data, float *main_data, float *temp_data, int stride, void *user_data), void *user_data, int w, int h, const char *fmt)
+int adm(int (*read_frame)(float *ref_data, float *main_data, float *temp_data, int stride, void *user_data), void *user_data, int w, int h)
```

Comment:
adm doesn't need `enum VmafPixelFormat fmt_enum`?

Reply:
yeah, the pixel format was not used inside there

Reply:
Feature extractors should just take a pair of `VmafPicture`s? `VmafPicture` is a data structure that holds all of this info already. If we need to add something like colorspace in the future, it just gets added to the `VmafPicture` struct and you don't need to go and add it as an additional function parameter for all N feature extractors.

Reply:
Yeah, we can propagate the changes from the API and the usage of VmafPicture internally. Perhaps we can make that change later and focus more on the API for now.